### DISCUSSION TITLE
Possibility to add discussions to Kanban Board?

### DISCUSSION THREAD

**Discussion by JustinBack** [community/community#3044] [Discussions]
We've recently enabled GitHub Discussions for our repositories, however one thing essential feature missing for us is adding discussions to a kanban board

![](https://justinback.jbcdn.net/captures/firefox_VDPpzsyEqG.png)

![](https://justinback.jbcdn.net/captures/firefox_5wd5Qb5mMI.png)

Is this feature planned/not going to be implemented/new? I haven't found anything regarding this yet and would love to see this implemented.

**Comment by becca** [comment]
Thank you so much for this feedback. We are tracking this feature request and will do our best to keep you posted on when this feature lands. Also, be sure to keep you eye on our [public roadmap](https://github.com/github/roadmap) for more updates!

**Comment by erikr** [comment]
Our team would love and regularly use this feature
